### PR TITLE
updated instructions

### PR DESCRIPTION
**Pull request by ibroom**
I need to also set the 'showOnPortalApps' to default to checked but I don't know how. It seems it will rarely not be checked.

### CODE REVIEW

File: interface.html [hunk 1]
```diff
@@ -186,8 +199,9 @@
             <div class="checkbox checkbox-icon">
               <input type="checkbox" id="portal-popup" name="showOnceOnPortal" value="1" {{#if showOnceOnPortal}}checked{{/if}} />
               <label for="portal-popup">
-                <span class="check"><i class="fa fa-check"></i></span> Only show once on portal apps
+                <span class="check"><i class="fa fa-check"></i></span> Apps inside a portal shouldn't ask request users subscibe to push notifications.
               </label>
+              <p class="help-block">TBD</p>
```
Comment (squallstar) [inline interface.html]: @ibroom do we need this "TBD"?

**Comment by squallstar** [comment]
Closed after inactivity.

**Comment by ibroom** [comment]
I think these changes are still required? I mainly suggested content, I didn't test or polish the UI.

**Comment by squallstar** [comment]
@ibroom this interface has gone through numerous changes over the year (your PR was made last August) and there's just too many conflicts between the current version and the file you originally edited. Furthermore, I strongly suggest simply documenting the required changes on a ticket so @hcarneiro or @tonytlwu or myself can take it over, rather than having to fix your HTML.

**Comment by tonytlwu** [comment]
Added proposed issue https://github.com/Fliplet/fliplet-studio/issues/2390 for further discussions.

**Pull request closed without merging**